chain-core/src/init/config.rs

```diff
@@ -2,85 +2,61 @@ use crate::common::Timespec;
 use crate::init::address::RedeemAddress;
```

Comment:
just one note that genesis processing may need more thought / design: https://github.com/crypto-com/chain-docs/issues/141#issuecomment-634387139